code-experiments/build/python/cython/interface.pyx

```diff
@@ -869,6 +872,16 @@ cdef class Problem:
     def best_observed_fvalue1(self):
         assert(self.problem)
         return coco_problem_get_best_observed_fvalue1(self.problem)
+    @property
+    def largest_fvalues_of_interest(self):
+        "largest f-values of interest (defined only for multi-objective problems)"
+        assert(self.problem)
+        if self._number_of_objectives > 1 and coco_problem_get_largest_fvalues_of_interest(self.problem) is not NULL:
+            # Not a particulary nice way of doing this, mostly due to cython
+            self._largest_fvalues_of_interest = np.inf * np.ones(self._number_of_objectives)
```

Comment:
Not clear why we need the `np.inf` here...

Reply:
We don't really, a `np.zeros` would probably suffice...

Reply:
Lines 880-883 could (should?) be replace by
```python
# calling coco_... several times should not be necessary, but it's not clear how to avoid it
self._largest_fvalues_of_interest = np.asarray(
        [coco_problem_get_largest_fvalues_of_interest(self.problem)[i]
         for i in range(self._number_of_objectives)])
```
This also eliminates the misleading initialization.

Reply:
Yes, this should do it...